mindsdb/integrations/handlers/openstreetmap_handler/openstreetmap_handler.py

```diff
@@ -0,0 +1,105 @@
+import overpy
+
+import pandas as pd
+from typing import Text, List, Dict
+
+from mindsdb.integrations.handlers.openstreetmap_handler.openstreetmap_tables import ( OpenStreetMapNodeTable, 
+    OpenStreetMapWayTable, OpenStreetMapRelationTable )
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+)
+
+
+class OpenStreetMapHandler(APIHandler):
+    """The OpenStreetMap handler implementation."""
+
+    def __init__(self, name: str, **kwargs):
+        """Registers all API tables and prepares the handler for an API connection.
+
+        Args:
+            name: (str): The handler name to use
+        """
+        super().__init__(name)
+
+        connection_data = kwargs.get("connection_data", {})
+        self.connection_data = connection_data
+        self.kwargs = kwargs
+
+        self.connection = None
+        self.is_connected = False
+
+        nodes_data = OpenStreetMapNodeTable(self)
+        self._register_table("nodes", nodes_data)
+
+        ways_data = OpenStreetMapWayTable(self)
+        self._register_table("ways", ways_data)
+
+        relations_data = OpenStreetMapRelationTable(self)
+        self._register_table("relations", relations_data)
+
+    def connect(self) -> StatusResponse:
+        """Set up the connection required by the handler.
+
+        Returns:
+            StatusResponse: connection object
+        """
+        if self.is_connected is True:
+            return self.connection
+
+        api_session = overpy.Overpass( **self.connection_data)
+
+        self.connection = api_session
+
+        self.is_connected = True
+
+        return self.connection
+    
+    def check_connection(self) -> StatusResponse:
+        """Check connection to the handler.
+
+        Returns:
+            HandlerStatusResponse
+        """
+        try:
+            self.connect()
+            return StatusResponse(status=True)
```

Comment:
@singh1203 In instantiating objects of the `HandlerStatusResponse` and `HandlerResponse`, you seem to be passing in a parameter called `status`. However, it does not look like either of these classes accept such a parameter. I am not able to execute `CREATE DATABASE` statements. as a result.